ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+While we invite contributions by companies or other organizations, contributors
+listed in a ITE need to be specific people.
+
+==== Sponsor
+
+Each ITE has at least one "Sponsor".
+
+The ITE Sponsor is a contributor who is responsible for the ITE throughout its
+lifecycle. Their responsibilities include:
+
+* Building consensus for that ITE within the community
+* Documenting dissenting opinions
+* Coordinating contributors' work
+* Ensuring the ITE meets the style, format, and quality guidelines
+* Maintaining the ITE after it is finalized
+* Setting and communicating the schedule as needed
+
+The Sponsor of a ITE may or may not do any of the tasks other contributors do.
+For example, one sponsor might write large portions of one ITE, while another
+sponsor might leave the writing to other contributors.
+
+Anyone may be Sponsor for a ITE, though it should be someone familiar enough
+with in-toto, the in-toto project, and the ITE workflow to effectively guide
+the ITE to completion.
+
+A ITE may have more than one Sponsor, especially after it has been finalized
+and is being maintained over time. For simplicity, this document uses the
+singular ("The ITE Sponsor", "a sponsor") when referring the one or more people
+in the role of "Sponsor" of a ITE.
+
+Sponsors have committer/write access on the ITE repository, but should
+only approve and merge pull requests for ITEs to which they are assigned.
+
+==== Reviewer
+
+The ITE Reviewer is the contributor who will make the final decision whether to
+accept a ITE. In all cases where this document refers to the `Reviewer`, it
+means "an editor who will review this ITE."
+
+[[requirement-levels]]
+==== Must/Should/May
+
+ITE documents _must_ follow link:https://tools.ietf.org/html/rfc2119[RFC 2119]
+which defines key words to "indicate requirement levels". These are listed
+below:
+
+. **MUST** This word, or the terms "REQUIRED" or "SHALL", mean that the
+   definition is an absolute requirement of the specification.
+. **MUST NOT** This phrase, or the phrase "SHALL NOT", mean that the
+   definition is an absolute prohibition of the specification.
+. **SHOULD** This word, or the adjective "RECOMMENDED", mean that there
+   may exist valid reasons in particular circumstances to ignore a
+   particular item, but the full implications must be understood and
+   carefully weighed before choosing a different course.
+. **SHOULD NOT** This phrase, or the phrase "NOT RECOMMENDED" mean that
+   there may exist valid reasons in particular circumstances when the
+   particular behavior is acceptable or even useful, but the full
+   implications should be understood and the case carefully weighed
+   before implementing any behavior described with this label.
+. **MAY** This word, or the adjective "OPTIONAL", mean that an item is
+   truly optional.
+
+NOTE: When choosing to go counter to SHOULD or SHOULD NOT guidance,
+the reasons behind that choice SHOULD be documented.
+
+=== ITE Workflow
```

Comment:
As part of an ITE workflow section, it seems that for an ITE to be approved, it must contain both design AND implementation for a new feature. If one just wants to propose a new feature by describing the design from a specification p.o.v, but doesn't provide an implementation, that cannot be done through the ITE process.
This is not a major concern, I'm just wondering if this is what we want. In fact, reading the rest of this document, it seems that not all ITEs must have an implementation, but the ITE workflow section suggests they must have one.